cli/bench/types.go

```diff
@@ -0,0 +1,36 @@
+package bench
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/FZambia/tarantool"
+	"github.com/tarantool/cartridge-cli/cli/context"
+)
+
+type Results struct {
+	hanled_requests_count int
+	success_result        int
+	failed_result         int
+	duration              float64
+	requests_per_secornd  int
+}
+
+func print_results(results Results) {
+	fmt.Println("Results:")
+	fmt.Println("	Success operations: ", results.success_result)
+	fmt.Println("	Failed  operations: ", results.failed_result)
+	fmt.Println("	Request count: ", results.hanled_requests_count)
+	fmt.Println("	Time (seconds): ", results.duration)
+	fmt.Println("	Requests per second: ", results.requests_per_secornd, "\n")
+}
+
+type request_t func(ctx context.BenchCtx, connection_id int, request_id int, tarantool_connection *tarantool.Connection, space_name string, results *Results, wait_group *sync.WaitGroup, bench_active_flag *bool)
+
+type Operaiton struct {
```

Comment:
what "operation" are we talking about?

Reply:
Abstract operation, such as insertion, deletion, etc.

Reply:
Maybe ``Operation`` struct can have methods ``DeleteRequestsLoop``, ``InsertRequestsLoop``? I'm not sure, but think about it.

Reply:
Now this architecture is enough for a simple benchmark tool for a single tarantool. It is not yet known what will be required in the future for more complex scenarios